### PR TITLE
add terminal chat section

### LINKED ISSUE #7159: Add docs for terminal chat
https://code.visualstudio.com/docs/copilot/copilot-chat

### PR DESCRIPTION
**Pull request by meganrogge**
fixes [#7159](https://github.com/microsoft/vscode-docs/issues/7159)

### CODE REVIEW

File: docs/copilot/copilot-chat.md [hunk 1]
```diff
@@ -57,6 +57,12 @@ The following example shows the `preview` mode for inline chat.
 
 ![Copilot inline chat preview mode, showing the code changes side-by-side](images/copilot-chat/inline-chat-preview-mode.png)
 
+## Experimental terminal inline chat
+
+Inline chat for the terminal can be enbled with `terminal.integrated.experimentalInlineChat` and invoked in a terminal (`kb(inlinechat.start)`).
+
+The chat will use the ![@terminal](###terminal) chat participant, which has context about the integrated terminal shell and its contents.
```
Comment (meganrogge) [inline docs/copilot/copilot-chat.md:64]: I wanted to do `![`@terminal`]` here but it didn't work in the preview...

Comment (Tyriar) [reply docs/copilot/copilot-chat.md:64]: The `!` prefix means it's an image, that's probably what was causing the issue.

File: docs/copilot/copilot-chat.md [hunk 2]
```diff
@@ -57,6 +57,12 @@ The following example shows the `preview` mode for inline chat.
 
 ![Copilot inline chat preview mode, showing the code changes side-by-side](images/copilot-chat/inline-chat-preview-mode.png)
 
+## Experimental terminal inline chat
+
+Inline chat for the terminal can be enbled with `terminal.integrated.experimentalInlineChat` and invoked in a terminal (`kb(inlinechat.start)`).
```
Comment (Tyriar) [inline docs/copilot/copilot-chat.md:62]:
```suggestion
Inline chat for the terminal can be enabled with `terminal.integrated.experimentalInlineChat` and invoked in a terminal (`kb(inlinechat.start)`).
```

File: docs/copilot/copilot-chat.md (hunk 1 above)
Comment (Tyriar) [inline docs/copilot/copilot-chat.md:64]: I think the `#` fragment is wrong here

```suggestion
The chat will use the [`@terminal`](#_terminal) chat participant, which has context about the integrated terminal shell and its contents.
```

File: docs/copilot/copilot-chat.md [hunk 3]
```diff
@@ -57,6 +57,12 @@ The following example shows the `preview` mode for inline chat.
 
 ![Copilot inline chat preview mode, showing the code changes side-by-side](images/copilot-chat/inline-chat-preview-mode.png)
 
+## Experimental terminal inline chat
```
Comment (Tyriar) [inline docs/copilot/copilot-chat.md:60]: While this is closely related to inline chat, it feels wrong to feature this before the chat view. @ntrogh any thoughts on where we should stick this? We could also hold off on adding a section until it's non-experimental.

**Review by Tyriar — CHANGES_REQUESTED** [review]

File: docs/copilot/copilot-chat.md (hunk 3 above)
Comment (meganrogge) [reply docs/copilot/copilot-chat.md:60]: Yeah I agree. I wasn't sure about this.

Comment (ntrogh) [reply docs/copilot/copilot-chat.md:60]: We typically hold off on putting experimental features in the main docs.

**Pull request closed without merging**